### PR TITLE
SB-25552 - UCI GQL Deployment

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25552" title="SB-25552" target="_blank">StorySB-25552</a>  UCI - Deployment
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by shiva-rakshith** [comment]
@keshavprasadms Please review the PR.

File: ansible/roles/stack-sunbird/templates/gql.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
+ENV=prod
+DATABASE_DEBUG=true
+HASURA_GRAPHQL_ADMIN_SECRET=HaTUNsNSaU
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/gql.env:3]: No default passwords in public repos

File: kubernetes/pipelines/deploy_uci/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+               sh'git clean -fxd'
+            }  
+            checkout scm
+        }
+
+        stage('deploy') {
+            envDir = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-3].trim()
+	        module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
+	        jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
+		    values = [:]
```
Comment (keshavprasadms) [inline kubernetes/pipelines/deploy_uci/Jenkinsfile:24]: Please correct spacing of this line and lines above this

Comment (keshavprasadms) [reply kubernetes/pipelines/deploy_uci/Jenkinsfile:24]: Also why is this file included here? Its already part of the UCI PR right

Comment (shiva-rakshith) [reply kubernetes/pipelines/deploy_uci/Jenkinsfile]: No, this is part of this PR only. GQL does not have build and artifactupload jobs, It has only deploy job.
So geetha provided this jenkinsfile.

File: ansible/roles/stack-sunbird/templates/gql.env (hunk 1 above)
Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/gql.env:3]: I have removed it for now, we have few more keys that need to be included in private repo and we will add those variables here.

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**